src/components/TripPlanner/SightingsTable.tsx

```diff
@@ -1,31 +1,21 @@
-import { useEffect, useState } from "react";
-import { TripPlannerRequest, TripPlannerResponse } from "../../clients/apiClient";
-import { getSightingsListByLocation } from "../../clients/apiClient";
+import { TripPlannerResponse } from "../../clients/apiClient";
 import './SightingTable.scss';
 
 interface SightingTableProps {
-    latLon?: TripPlannerRequest;
+    listSightings: TripPlannerResponse[];
 }
 
-export function SightingsTable({ latLon }: SightingTableProps) {
-
-    const [listSightings, setListSightings] = useState<TripPlannerResponse[]>([]);
-    console.log(latLon?.latitude, latLon?.longitude);
-
-    useEffect(() => {
-        if (latLon) {
-            getSightingsListByLocation(latLon)
-                .then(response => { if (response) { setListSightings(response); console.log(response); } })
-        }
-    }, [latLon]);
+export function SightingsTable(props: SightingTableProps) {
```

Comment:
Can we change this back to the other syntax, then when you call this you will just use `listSightings` instead of `props.listSightings`
```suggestion
export function SightingsTable({listSightings}: SightingTableProps) {
```

Reply:
changed